### ISSUE TITLE
Build R2023a and all Nightly Build 2023: WeBots doesn't start (dll files were not found on windows 11)

### ISSUE THREAD

**Issue by ali1hammoud** [cyberbotics/webots#6078]
After installing webots 2023 and trying all versions, for Windows, in the form of EXE installer,
WeBots doesn't start because of a missing DLL, libgcc_s_seh-1.dll, libstdc++-6.dll, libwinpthread-1.dll

Operating System: Windows 10

**Comment by BenjaminDeleze** [comment]
I tried to install R2023a and the latest Nightly Build both on Windows 10 and Windows 11 and got no problem.
It may be possible that the problem is coming from your system.

**Comment by ali1hammoud** [comment]
Thank you for your answer.
I downloaded these file from internet ( libgcc_s_seh-1.dll, libstdc++-6.dll, libwinpthread-1.dll), and added them to this path:
C:\Program Files\Webots\msys64\mingw64\bin

but I got these errors:
![image](https://user-images.githubusercontent.com/37132652/232462184-ac2190d9-1b7d-4bc0-a328-8f51a6fb3946.png)

![image](https://user-images.githubusercontent.com/37132652/232462488-20f25b71-5630-4d0e-b6e7-f46fb9681f96.png)

![image](https://user-images.githubusercontent.com/37132652/232462584-c7c3f8e1-aa83-4e19-9ab0-06a076d66291.png)

![image](https://user-images.githubusercontent.com/37132652/232462684-dc1e896e-72b2-4840-b180-9481ab3bdd3b.png)

are there any ideas where the problem could be?

**Comment by BenjaminDeleze** [comment]
@omichel do you have an idea?

**Comment by omichel** [comment]
No idea. But it is probably a bad idea to download libraries from the Internet as they are likely not the same version as the ones supposed to be included within Webots. These 3 libraries are normally installed by Webots inside the `C:\Program Files\Webots\msys64\mingw64\bin\cpp` folder. Don't you have them here?

**Comment by ali1hammoud** [comment]
Yes these files inside the `C:\Program Files\Webots\msys64\mingw64\bin\cpp` folder, but I don't know why I got that error.

I copied the files to this folder:   `C:\Program Files\Webots\msys64\mingw64\bin`, and now I have this error:
![image](https://user-images.githubusercontent.com/37132652/232729239-5488ae8d-c751-4a2e-ac3a-07c3c65944ea.png)

**Comment by omichel** [comment]
Did you try to launch Webots from the icon (instead of launching it from the DOS console)?

**Comment by ali1hammoud** [comment]
Yes, and nothing is showing.

in task manager, I see that webots start for few seconds then the process end immediately without any GUI.

**Comment by omichel** [comment]
I have no more idea... That could be a problem with your local firewall...
